src/NodaTime/Text/PeriodPattern.cs

```diff
@@ -184,6 +184,14 @@ public StringBuilder AppendFormat(Period value, StringBuilder builder)
             {
                 Preconditions.CheckNotNull(value, nameof(value));
                 Preconditions.CheckNotNull(builder, nameof(builder));
+                // Always ensure we've got *some* unit; arbitrarily pick days.
```

Comment:
"... to ensure that the result is valid per ISO 8601" or similar?

Reply:
Done.